integration/yarn/src/main/java/alluxio/yarn/ApplicationMaster.java

```diff
@@ -64,24 +63,14 @@
 public final class ApplicationMaster implements AMRMClientAsync.CallbackHandler {
   private static final Logger LOG = LoggerFactory.getLogger(Constants.LOGGER_TYPE);
 
-  /** This tarball is generated by alluxio-yarn.sh. */
-  private static final String ALLUXIO_TARBALL = "alluxio.tar.gz";
-  /**
-   * Because our jars are available as local resources in the working directory from which the
-   * command will be run, we need to append "." to the path.
-   */
-  private static final String ENV_CLASSPATH =
-      new StringBuilder(ApplicationConstants.Environment.CLASSPATH.$())
-          .append(File.pathSeparatorChar).append("./*").toString();
   /** Maximum number of rounds of requesting and re-requesting worker containers. */
-  // TODO(andrew): make this configurable
```

Comment:
It's not clear whether it will ever be useful to make it configurable, so I don't think it's important enough to warrant a TODO